RecoTracker/FinalTrackSelectors/python/SingleLongTrackProducer_cfi.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoTracker.FinalTrackSelectors.singleLongTrackProducer_cfi import singleLongTrackProducer 
+
+SingleLongTrackProducer = singleLongTrackProducer.clone(
+    allTracks = "generalTracks",
+    matchMuons = "muons",
```

Comment:
```suggestion
    matchMuons = "earlyMuons",
```

as privately discussed a possible way forward to avoid to have to deal with fully reconstructed events in the case of the `trackingOnly` DQM sequence is to use the `earlyMuons` collection which is used as input of the muon-seeded iterations in the iterative tracking configuration. 
I wrapped that (along with other improvements in [this commit](https://github.com/cms-sw/cmssw/commit/524f5809e577c54123a8b74fb047876e1707b5c5) ). Feel free to apply (or close the PR and use the branch above).